python/ray/dag/compiled_dag_node.py

```diff
@@ -681,13 +684,10 @@ def __init__(
         # Mapping from the actor handle to the node ID that the actor is on.
         self.actor_to_node_id: Dict["ray.actor.ActorHandle", str] = {}
 
-        # This is set to true when type hint of `transport="nccl"` is used.
-        self._use_default_nccl_group = False
-        # This is set to the specified custom nccl group
-        # if there exists a type hint of `transport=nccl_group`.
-        self._custom_nccl_group_p2p: Optional[GPUCommunicator] = None
-        # The NCCL group ID for P2P send/recv operations.
-        self._nccl_group_id_p2p: Optional[str] = None
+        # The `custom_nccl_group` specified in `experimental_compile`.
```

Comment:
Optional: Make these lines of comments similar.

Reply:
These variable names are very similar. You should explain the differences clearly.

Reply:
Named both `default_`. Removed the difference between them. Now the id is always the id of the group (if the group is initialized).